src/adapters/github.js

```diff
@@ -205,11 +206,81 @@ class GitHub extends Adapter {
   _getTree(path, opts, cb) {
     this._get(`/git/trees/${path}`, opts, (err, res) => {
       if (err) cb(err)
-      else cb(null, res.tree)
+      else {
+        const diff = this._getPatch(path)
+        const diffExists = (Object.keys(diff).length > 0)
+        const filteredTree = res.tree.filter((node) => {
+          node.patch = diff[node.path]
+          delete diff[node.path]
+          return !diffExists || node.patch
+        })
+
+        cb(null, filteredTree)
+      }
     })
   }
 
   // @override
+   _getPatch(path) {
+     const diff = {}
+     const files = $(".diff-view .file-info")
+     files.each(function() {
+       const file = $(this).find("a.link-gray-dark").first()
+
+       let path = file.attr("title")
+       let href = file.attr("href")
+       let previous = ""
```

Comment:
Please use single quotes always. Also others: arrow instead of func, else in new line, === and !== instead of == and !=.